lab-anthony/app/components/app-gallery/app-gallery.html

```diff
@@ -0,0 +1,20 @@
+<section class="app-gallery">
+  <div class="list-view">
+    <h2>List View</h2>
+    <ul>
+      <app-gallery-text ng-repeat="item in appGalleryCtrl.images" title="{{ item.title }}" desc="{{ item.desc}}" img-src="{{ item.imgSrc }}" ></app-gallery-text>
```

Comment:
Why are you passing in all of the properties on the image object separately instead of just passing the whole object through? This way is less extensible, more likely to have problems with angular, and requires more typing. I would do it like:

```html
<app-gallery-text data-ng-repeat="item in appGalleryCtrl.images" image="item"></app-gallery-text>
```